### ISSUE TITLE
lume -s, Fail to copy file to where the path once was a directory

### ISSUE THREAD

**Issue by iacore** [lumeland/lume#626]
### Version

about the latest

### Platform

Linux

### What steps will reproduce the bug?

1. `site.copy('public')`
2.
```
deno task serve &
# wait for server to launch
cd public/
mkdir b.md
touch b.md/b.md
rm -r b.md; touch b.md
```

### How often does it reproduce? Is there a required condition?

100% methinks

### What is the expected behavior?

There should be no error.

### What do you see instead?
```
ERROR Failed to copy file: /b.md/b.md: Not a directory (os error 20): readfile '/home/user/__site__/public/b.md/b.md'
```
### Additional information

workaround: run `deno task serve` again.

**Comment by oscarotero** [comment]
Do you mean this error should be cached and showed in the terminal but without exit?

**Comment by iacore** [comment]
> Do you mean this error should be cached and showed in the terminal but without exit?

ohhh sorry i forgot to fill in the field; it should not be an user-facing error.

**Comment by oscarotero** [comment]
This has been fixed and will be released in Lume 2.3.
From now on, if a file is removed and its folder is empty, it's removed too.
